developer/src/kmc-kmn/src/kmw-compiler/kmw-compiler.ts

```diff
@@ -228,33 +218,32 @@ export function WriteCompiledKeyboard(
   const isDebugStore = (index: number) => isStoreType(index, STORETYPE_DEBUG);
   const isReservedStore = (index: number) => isStoreType(index, STORETYPE_RESERVED);
   const isOptionStore = (index: number) => isStoreType(index, STORETYPE_OPTION);
+  const getStoreLine = (index: number) => kmxResult.extra.stores[index].line;
 
 	// Write the stores out
   FOptionStores = '';
 	for(let i = 0; i < keyboard.stores.length; i++) {
     let fsp = keyboard.stores[i];
     // I3438 - Save all system stores to the keyboard, for now   // I3684
 
-    if (!isDebugStore(i)) { // and not (fsp.dwSystemID in [TSS_BITMAP, TSS_NAME, TSS_VERSION, TSS_CUSTOMKEYMANEDITION, TSS_CUSTOMKEYMANEDITIONNAME, TSS_KEYMANCOPYRIGHT]) then
```

Comment:
How did the `// and not ...` get resolved here?

Reply:
That was legacy commented code -- it was already a very old comment in the KeymanWebCompiler.pas source, so safe to delete here.